.github/workflows/rust.yml

```diff
@@ -195,7 +203,7 @@ jobs:
         toolchain: ${{ fromJSON(needs.setup.outputs.toolchains) }}
         flags:
```

Comment:
I think this workflow file is becoming more difficult to understand as well. The flags here are pretty strange at first glance, seems like we're doing `cargo run --all-features --no-default-features`